docs/arcadia-users-group/20240305-github-templates/lesson.md

```diff
@@ -0,0 +1,88 @@
+# Introduction to GitHub Templates
+
+GitHub templates provide a structured way to create new repositories, ensuring consistency, efficiency, and adherence to best practices across projects.
+GitHub templates are predefined repository setups that can be used to start new projects with a consistent set of files, folders, and configurations.
+This not only saves time but helps make sure that all projects follow Arcadia's coding and documentation standards.
+Templates typically include everything from README files, installation guides, contribution guidelines, and more.
+
+In this lesson, we will focus on Arcadia's [`python-analysis-template`](https://github.com/Arcadia-Science/python-analysis-template).
+Arcadia also has an [`r-analysis-template`](https://github.com/Arcadia-Science/r-analysis-template), a [`snakemake-template`](https://github.com/Arcadia-Science/snakemake-template), and a [`nextflow-template`](https://github.com/Arcadia-Science/nextflow-template).
+
+## Creating a new repository using a template
+
+### Choosing the Right Template
+
+Depending on the analysis type or the technology stack used in your project, select the appropriate template. 
+For instance, use `r-analysis-template` for R-based analyses, `python-analysis-template` for Python projects, `snakemake-template` for workflows managed with Snakemake, and `nextflow-template` for Nextflow pipelines.
+
+<details>
+<summary>Why isn't there a Python package template?</summary>
+
+We could create one. 
+We haven't generated one yet because there are a ton templates out there that could be used and we haven't created many Python packages yet as an organization.
+If you find yourself in need of a Python template, try out <a href=https://github.com/TezRomacH/python-package-template>an existing template</a> and make note of what works well and what doesn't.
```

Comment:
should we note here that people can also ask the software team for help with this (or is that implied/understood)?